.github/workflows/check-on-push.yml

```
@@ -23,3 +23,15 @@ jobs:
         env:
           TAURI_DIST_DIR: ${{ runner.workspace }}/tauri/tauri/test/fixture/dist
           TAURI_DIR: ${{ runner.workspace }}/tauri/tauri/test/fixture/src-tauri
+
+  eslint-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1.4.0
```

```suggestion
      - uses: actions/setup-node@v1
```
I think we can safely use just the major version. They are updating this tag properly it seems that it will always be on the latest of v1. It will save us from having to monitor and upgrading it.

---

Ah, I see. I assumed that meant v1.0.0.